app/views/exports/hamilton_lrc_pbmc_defrost_pbs_to_lrc_pbmc_pools.csv.erb

```diff
@@ -0,0 +1,58 @@
+<%# This is a picking file for a Hamilton robot for LRC PBMC Defrost plate to LRC PBMC Pools plate transfers %>
+<%= CSV.generate_line [
+    'Workflow', 
+    @workflow
+  ], 
+  row_sep: '' 
+%>
+<%= CSV.generate_line [
+    'SourcePlate',
+    'SourceWell',
+    'DestinationPlate',
+    'DestinationWell',
+    'SampleVolume',
+    'ResuspensionVolume',
+  ],
+  row_sep: ''
+%>
+<%
+  # Prepare the fields determined from the transfer requests
+
+  required_number_of_cells = 5_000
+  maximum_sample_volume = 50.0  # microlitres
+  resuspension_volume_per_sample = 2.2  # microlitres
+  minimum_resuspension_volume = 10.0  # microlitres
+  millilitres_to_microlitres = 1_000.0
+
+  # Make a mapping { src_barcode => { src_location => src_well } }
+  ancestor_plates_wells = @ancestor_plate_list.each_with_object({}) do |ancestor_plate, hash|
+    hash[ancestor_plate.labware_barcode.human] = ancestor_plate.wells.index_by(&:location)
+  end
+
+  transfer_request_data = []
+  each_source_metadata_for_plate(@plate) do |src_barcode, src_location, dest_well|
+    src_well = ancestor_plates_wells[src_barcode][src_location]
+    cell_count = src_well.latest_live_cell_count  # cells / millilitres
+    next if cell_count.nil?
```

Comment:
There seems to be a bit of an inconsistency here, because the LIMS will look as if that sample was transferred, right? Because the sample is already present in the destination well?

Not sure how this should behave, if the cell count data is not present.

Reply:
I don't think there is an inconsistency. The plate will be created. However, driver file download means samples are not transferred yet. Bed-verification will be done (when the story is done) and transfer will be started. 

We did not include the count data in the transfer request. We collect it from source wells. Whenever the QC data are available upstream, a fresh new driver file download will include the latest QC. Are you suggesting that we should have some kind of verification?

At the moment, I am skipping the rows if cell count is not available, similar to Cardinal.